[onboarding] Clarify how emoji are currently accepted in update and create endpoints (#6479)
hammerandchisel/discord-api-docs: docs/resources/Guild.md

###### Prompt Option Structure

| Field           | Type                                               | Description                                                       |
|-----------------|----------------------------------------------------|-------------------------------------------------------------------|
| id              | snowflake                                          | ID of the prompt option                                           |
| channel_ids     | array of snowflakes                                | IDs for channels a member is added to when the option is selected |
| role_ids        | array of snowflakes                                | IDs for roles assigned to a member when the option is selected    |

> | emoji?          | [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | Emoji of the option (see info)                                    |

Reviewer: ```suggestion
| emoji?          | [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | Emoji of the option (see below)                                   |
```

Reviewer: I don't love referencing things outside of the table like this, but if we do I think "below" or "the callout below" something makes more sense than "info"

> | emoji_id?       | snowflake                                          | Emoji ID of the option (see info)                                 |

Reviewer: ```suggestion
| emoji_id?       | snowflake                                          | Emoji ID of the option (see below)                                |
```

> | emoji_name?     | string                                             | Emoji Name the option (see info)                                  |

Reviewer: ```suggestion
| emoji_name?     | string                                             | Emoji name of the option (see below)                              |
```

> | emoji_animated? | boolean                                            | Is the emoji animated (see info)                                  |

Reviewer: ```suggestion
| emoji_animated? | boolean                                            | Whether the emoji is animated (see below)                         |
```

| title           | string                                             | Title of the option                                               |

> | description     | ?string                                            | Description of the option                                         |
> > info

Reviewer: ```suggestion
| description     | ?string                                            | Description of the option                                         |

> warn
```

> > The emoji object is returned, but we expect the flattened `emoji_id`, `emoji_name`, `emoji_animated` when creating or updating the prompt option.

Reviewer: ```suggestion
> When creating or updating a prompt option, the `emoji_id`, `emoji_name`, and `emoji_animated` fields must be used instead of the emoji object.
```
